artifactory/cli.go

```diff
@@ -952,6 +952,10 @@ func getSearchFlags() []cli.Flag {
 			Name:  "build",
 			Usage: "[Optional] If specified, only artifacts of the specified build are matched. The property format is build-name/build-number. If you do not specify the build number, the artifacts are filtered by the latest build number.` `",
 		},
+		cli.BoolFlag{
+			Name:  "count",
+			Usage: "[Optional] Set to true if you do not wish to print the entire JSon result but only the total of found artifacts.` `",
```

Comment:
I suggest changing the description to:
*Set to true to display only the total of files or folders found.*

Reply:
Done